### PR TITLE
Added Head Imgae; header sign-up link; 

### PR DESCRIPTION
**Pull request by rainyuxuan**
ready to review

### CODE REVIEW

**Comment by Falanke21** [comment]
image

There's a small white area near the bottom of my screen. 
Do you want a whole screen span? If so, use width: `100vw; height: 100vh` in css. 
It will work much better than using % sign

File: index.css [hunk 1]
```diff
@@ -8,22 +8,118 @@ body {
...
+
+/* Head Image section */
+
+#headImageSection {
+    background: black;
+    backdrop-filter: blur(50px);
+}
+
+#headImage {
+    width: 100%;
```
Comment (Falanke21) [inline index.css]: change to vh here :)

**Comment by Falanke21** [comment]
Just a small suggestion, I don't think Jin will like the `2020` design lol

**Comment by rainyuxuan** [comment]
resolved, ready to review

**Comment by Falanke21** [comment]
image

image

Is this a desired behaviour?

**Comment by rainyuxuan** [comment]
不ready了，new design打回原厂

**Pull request closed without merging**